### PR TITLE
fixes #805 Persistent dismissal of image upload message

### LINKED ISSUE #805: Persistent dismissal of image upload message
**Summary:** 

The message that informs the user that only one image is necessary for an upload should not be displayed once the user dismisses this. You can check out the message that was added in #789. The message should always appear, but once dismissed by the user, this should be stored in prefs and never shown again.

**Expected behavior:** 

This should always display unless dismissed by the user. Once dismissed by the user, it should never show again.

### PR DESCRIPTION
**Pull request by Shubham2320**
## Description

SharedPreference is used to store the boolean (message is dismissed or not).

## Related issues and discussion
fixes #805

### CODE REVIEW

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/SaveProductOfflineActivity.java [hunk 1]
```diff
@@ -88,7 +103,11 @@ protected void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
         mSendProductDao = Utils.getAppDaoSession(this).getSendProductDao();
-
+        mSharedPref = getApplicationContext().getSharedPreferences("prefs", 0);
+        boolean messageDismissed = mSharedPref.getBoolean("is_message_dismissed", false);
```
Comment (teolemon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/SaveProductOfflineActivity.java]: Can we be much more explicit about the message ? I expect that eventually we'll have many explainer messages at many places in the app.

**Comment by Shubham2320** [comment]
@teolemon I've made the changes please review :).

**Comment by teolemon** [comment]
Is a migration script necessary for the ORM ? @itchix ?

**Comment by Shubham2320** [comment]
Are there any more changes which I need to do?

**Comment by itchix** [comment]
@teolemon I don't think so
He use `getSharedPreferences` so there is no modification in the ORM

**Comment by teolemon** [comment]
perfect, let's merge. thanks @Shubham2320

**Pull request merged**